app/workers/articles/bust_multiple_caches_worker.rb

```diff
@@ -0,0 +1,14 @@
+module Articles
+  class BustMultipleCachesWorker
+    include Sidekiq::Worker
+    sidekiq_options queue: :default, retry: 10
+
+    def perform(article_ids, cache_buster = "CacheBuster")
+      buster = cache_buster.constantize
```

Comment:
I think we can remove `cache_buster` as an argument and just use `CacheBuster.bust` in the job, right @mstruve ?